aten/src/ATen/core/jit_type.h

```diff
@@ -1145,7 +1145,7 @@ struct CAFFE2_API EnumType : public NamedType {
         AT_ERROR(
             "Cannot create Enum with value type '",
             value->str(),
-            "', only int, float, Tensor and string keys are supported");
+            "', only int, float and string are supported");
```

Comment:
We're not including bool  as a type here, but we are registering a kernel for it